src/main/java/org/example/Person.java

```diff
@@ -44,8 +56,43 @@ public void decreaseHealth(int value) {
             this.health = 0;
         }
     }
+    public void increaseMoney(int value) {
+        if (this.money + value <= 10000) {
+            this.money += value;
+        } else {
+            this.money = 10000;
+        }
+    }
+    public void decreaseMoney(int value) {
+        if (this.money - value >= 0) {
+            this.money -= value;
+        } else {
+            this.money = 0;
+        }
+    }
+    public void increaseSocial(int value) {
+        if (this.social + value <= 1000) {
+            this.social += value;
+        } else {
+            this.social = 1000;
+        }
+    }
+    public void decreaseSocial(int value) {
+        if (this.social - value >= 0) {
+            this.social -= value;
+        } else {
+            this.social = 0;
+        }
+    }
+
     public void increaseSkill(String skill){
```

Comment:
increaseSkill powinno zmieniać tylko skille , zmiana innych właściwości powinna byc oddelegowana do klasy, która operuje na Person, W tym programie są to eventy